server.js

```diff
@@ -50,17 +51,30 @@ app.use(bodyParser.urlencoded({
   extended: true
 }));
 
-// send assetPath to all views
+// Support for session
+app.use(session({secret: "prototype-kit"}));
+
 app.use(function (req, res, next) {
+
+  // store any data sent in session
+  for (var i in req.body){
+    req.session[i] = req.body[i];
+  }
+
+  // send session data to all views
+  for (var i in req.session){
+    res.locals[i] = req.session[i];
```

Comment:
I wonder if the automatic data should be saved in `res.locals.session`? That would prevent the data being clobbered any other local variables (or vice versa). Would also make it easier to debug/understand.